properties/P000166.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: P000166
+name: Has a coarser separable metrizable topology
+aliases:
+- Has a separable metrizable compression
+refs:
+- doi: 10.1007/BFb0098389
+  name: Topological Properties of Spaces of Continuous Functions
+---
+
+The topology contains a coarser topology which is both {P26} and {P53}.
+
+N.B. if the space is also {P6}, by {{doi:10.1007/BFb0098389}} this is equivalent to
```

Comment:
Is there a specific result we could point to from that reference?

Reply:
Ah I got away with this in my paper, but you're right, it should be pointed out more carefully. I'm on campus and hopefully can look this up...

Reply:
It seems we have a hard copy but getting to the library isn't in scope for my day today. I'm going to open an issue to improve this when I can.